scripts/check_configs.ts

```diff
@@ -426,6 +443,35 @@ function isValidMarketPromo(marketPromo: {[marketId: string]: MarketPromo}, netw
   return true;
 }
 
+function isValidTransferBanner(transferBanner: TransferBanner, network: CarbonSDK.Network): boolean {
+  const { temporary_disabled_transfer_tokens, temporary_disabled_bridges } = transferBanner;
+  if (Object.keys(temporary_disabled_transfer_tokens).length > 0) {
+    Object.keys(temporary_disabled_transfer_tokens).map((key) => {
```

Comment:
1. I see `Object.keys(temporary_disabled_transfer_tokens)` is used twice, pls store this in a variable and apply where necessary.
i.e.
```typescript
const disabledTokenKeys = Object.keys(temporary_disabled_transfer_tokens)
if (disabledTokenKeys.length > 0) {
  disabledTokenKeys.forEach((key) => { ... })
}
```

2. Pls check that each denom key is valid. Please refer to this [code snippet](https://github.com/Switcheo/demex-webapp-config/blob/master/scripts/check_configs.ts#L251) to see how to check if denoms are valid.